### ISSUE TITLE
Import of dataclasses_json under Python 3.6 raises a TypeError: metaclass conflict

### ISSUE THREAD

**Issue by jmfarrow** [lidatong/dataclasses-json#95]
Importing dataclasses_json 0.2.5 in Python 3.6 throws a metaclass conflict related `TypeError`

```
$ python3.6
Python 3.6.7 (default, Oct 22 2018, 11:32:17)
[GCC 8.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import dataclasses_json
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../lib/python3.6/site-packages/dataclasses_json/__init__.py", line 1, in <module>
    from dataclasses_json.api import (DataClassJsonMixin,
  File ".../lib/python3.6/site-packages/dataclasses_json/api.py", line 5, in <module>
    from dataclasses_json.mm import build_schema, SchemaHelper, JsonData
  File ".../lib/python3.6/site-packages/dataclasses_json/mm.py", line 105, in <module>
    class SchemaHelper(Schema, typing.Generic[T]):
TypeError: metaclass conflict: the metaclass of a derived class must be a (non-strict) subclass of the metaclasses of all its bases```

**Comment by jmfarrow** [comment]
Also under 3.6.8:
```
Python 3.6.8 (default, May 29 2019, 17:50:11)
[GCC 4.2.1 Compatible Apple LLVM 10.0.1 (clang-1001.0.46.4)] on darwin```

**Comment by lidatong** [comment]
hi, thanks for raising this issue. addressed by #96 -- please try the latest release (0.2.7)

**Comment by jmfarrow** [comment]
0.2.7 Works For Me™ under 3.6. Thanks.
